tams/data.py

```diff
@@ -66,32 +66,91 @@ def tb_from_ir(r, ch: int):
+    * Satellite data (EUMETSAT IR radiance):
+      https://drive.google.com/file/d/1HAhAlfqZGjnTk8NAjyx_lmVumUu_1TMp/view?usp=sharing
+    * MPAS regridded data (brightness temperature and precipitation):
+      https://drive.google.com/file/d/1vtx6UeSS8FM5Hy9DEQe3x78Ey-Hn-83E/view?usp=sharing
+    * MPAS native output (unstructured grid) data (brightness temperature and precipitation):
+      https://drive.google.com/file/d/1bexeAGSzS3FPEy3a120Z2Qsz0LC5_HPf/view?usp=sharing
 
+    Parameters
+    ----------
+    gdown
+        Use gdown. Otherwise, use wget.
+
+        .. note::
+           `gdown <https://github.com/wkentaro/gdown>`__ is not currently
+           a TAMS required dependency, so you must install it
+           in order to use ``gdown=True``.
+           It is available on conda-forge and PyPI as ``gdown``.
+    clobber
+        If set, overwrite existing files. Otherwise, skip downloading.
+
+    See Also
+    --------
+    tams.load_example_tb
+    tams.load_example_mpas
+    tams.load_example_mpas_ug
+    """
+
+    files = [
+        ("1HAhAlfqZGjnTk8NAjyx_lmVumUu_1TMp", "Satellite_data.nc"),  # < 100 MB
+        ("1vtx6UeSS8FM5Hy9DEQe3x78Ey-Hn-83E", "MPAS_data.nc"),  # < 100 MB
+        ("1bexeAGSzS3FPEy3a120Z2Qsz0LC5_HPf", "MPAS_unstructured_data.nc"),  # > 100 MB
+    ]
+
+    if gdown:
         try:
-            subprocess.run(cmd)
-        except Exception:
-            print(f"Running\n  {' '.join(cmd)}\nfailed:")
-            raise
+            import gdown as gdown_mod
+        except ImportError as e:
+            raise RuntimeError("gdown required") from e
+
+        def download(id_: str, to: str):
+            gdown_mod.download(id=id_, output=to, quiet=False)
+
+    else:
+        import subprocess
+
+        def download(id_: str, to: str):
+            url = f"https://drive.google.com/uc?export=download&id={id_}&confirm=t"
+            cmd = [
+                "wget",
+                "--no-verbose",
+                "--no-check-certificate",
+                url,
+                "-O",
+                to,
+            ]
+            try:
+                subprocess.run(cmd)
+            except Exception:
+                print(f"Running\n  {' '.join(cmd)}\nfailed:")
+                raise
+
+    for id_, fn in files:
+        fp = HERE / fn
+        if not clobber and fp.is_file():
+            print(f"Skipping {fn} because it already exists at {fp.as_posix()}.")
+            continue
+        else:
+            download(id_, fp.as_posix())
 
 
 def load_example_ir() -> xarray.DataArray:
     """Load the example satellite IR radiance data (ch9) as a DataArray.
 
     This dataset contains 6 time steps of 2-hourly data (every 2 hours):
     2006-09-01 00--10
+
+    See Also
+    --------
+    tams.data.download_examples
     """
 
-    ds = xr.open_dataset(HERE / "Satellite_data.nc").rename_dims(
+    ds = xr.open_dataset(HERE / "Satellite_data.nc", lock=False).rename_dims(
```

Comment:
```suggestion
    ds = xr.open_dataset(HERE / "Satellite_data.nc").rename_dims(
```
Just need dask-core, or new-enough xarray that bundles the SerializableLock (v2024.01.1).